multiworld/envs/mujoco/__init__.py

```diff
@@ -18,359 +18,207 @@ def register_custom_envs():
     """
     Reaching tasks
     """
+
     register(
         id='SawyerReachXYEnv-v0',
         entry_point='multiworld.envs.mujoco.sawyer_xyz.sawyer_reach:SawyerReachXYEnv',
         tags={
-            'git-commit-hash': 'c5e15f7',
-            'author': 'vitchyr'
+            'git-commit-hash': '2d95c75',
+            'author': 'murtaza'
         },
         kwargs={
-            'hide_goal_markers': False,
+            'hide_goal_markers': True,
+            'norm_order':2,
         },
     )
+
     register(
         id='Image48SawyerReachXYEnv-v0',
         entry_point=create_image_48_sawyer_reach_xy_env_v0,
         tags={
-            'git-commit-hash': 'c5e15f7',
+            'git-commit-hash': '2d95c75',
```

Comment:
The whole point of saving the git commit hash is to say "this was the state of the code when this env was made." If you want to update an env, you can just make a new version.